sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -28,6 +28,7 @@
 #include "vtimer.h"
 
 #include "net/ng_ndp.h"
+#include "net/ng_fib.h"
```

Comment:
The linker will complain about unresolved symbols. Those you can guard with `#ifdef MODULE_FIB`